web/packages/teleterm/src/ui/ModalsHost/modals/HardwareKeys/AskPin.tsx

```diff
@@ -63,20 +65,15 @@ export function AskPin(props: {
 
             <DialogContent mb={4}>
               <Flex flexDirection="column" gap={4} alignItems="flex-start">
```

Comment:
@gzdunek @ravicious I had to use Flex instead of Stack for this v16 backport since https://github.com/gravitational/teleport/pull/52003 wasn't backported. I tested it and it looks fine to me, but please let me know if it needs adjustments.

Reply:
> We also need to wrap some elements in OnlineDocumentGateway in Flex, the same way it was done in https://github.com/gravitational/teleport/commit/f8bc47b666de612e7e6ab60fd6528c1572dccffe.

Sorry, I'm not following the suggestion. Which elements do I need to add in the Flex?

Reply:
I meant wrapping the form, CliCommand and errors. I pushed the fix in https://github.com/gravitational/teleport/pull/54298/commits/9bf9bdb81ec4a6828b9fb7e91a7a686a15b38592.

I also decreased the gap size in `AskPin.tsx` and `Touch.tsx`.

Reply:
At this point it's too late, but I have a v16 backport for Stack in #52227. Bartosz asked me about it one time and then he didn't need it anymore, hence why it's a draft.

Reply:
If we can get it merged by tomorrow I can rebase it into this PR before I open it